src/main/java/com/byznass/tiolktrack/config/PropertyProvider.java

```diff
@@ -0,0 +1,15 @@
+package com.byznass.tiolktrack.config;
+
+public class PropertyProvider {
+
+	public String getValue(Property property) throws NoSuchPropertyException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.config.NoSuchPropertyException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)